packages/color-modes/test/custom-properties.tsx

```diff
@@ -201,6 +202,31 @@ describe('__createColorStyles', () => {
     })
   })
 
+  test('creates styles for print color mode if its name is the same as initialColorModeName', () => {
+    const styles = __createColorStyles({
+      config: {
+        initialColorModeName: 'light',
+        useColorSchemeMediaQuery: true,
+        printColorModeName: 'light',
+      },
+      colors: {
+        text: '#fff',
+        background: '#000',
+        modes: {
+          light: {
```

Comment:
This is a theme _after mutation_. Previous code assumed initial color mode is always at the root.